package.json

```diff
@@ -35,6 +35,7 @@
     "react": "0.14.x || 15.x.x"
   },
   "devDependencies": {
+    "babel-eslint": "^7.1.0",
```

Comment:
because Eslint is configured to use babel-eslint as parser and many IDE (such as Atom) require this package to be installed locally.